plugins/SimpleNtupler_miniAOD.cc

```diff
@@ -236,6 +236,11 @@ class SimpleNtupler_miniAOD : public edm::EDAnalyzer {
     float jet_eta[10];
     float jet_phi[10];
     float jet_btag[10];
+    //Added jet information
+    float jet_px[10];
```

Comment:
Don't you already have pT, eta, and phi? Then all you'd need to add would be E for a full TLorentzVector filling.
There are several methods for that, from SetPtEtaPhiE over SetPxPyPzE to SetPtEtaPhiM